### PR TITLE
remove outdated push statement

### LINKED ISSUE #462: faq: wording of background notification service usage outdated
we technically use APN for the heartbeat-wakeup

### PR DESCRIPTION
**Pull request by r10s**
closes #462

### CODE REVIEW

**Comment by missytake** [comment]
The "However" in the next sentence is still weird...

**Comment by r10s** [comment]
> The "However" in the next sentence is still weird...

any suggestions?

i still find it somehow fitting, at least not more weird as before :)

File: en/help.md [hunk 1]
```diff
@@ -444,9 +444,6 @@ Otherwise, deleting messages or multi-device setups might not work properly.
 - Receiving messages then can take minutes because both Android and iOS often
   stop Delta Chat from running in the background, and only wake it up
   occasionally. This artificial delay is usually worse on iOS than on Android.
-- Note that Delta Chat doesn't use Google Cloud Messaging (GCM) or the Apple
-  Push Notification Service (APNS), because this leads to user tracking and
-  central control which Delta Chat aims to avoid as much as feasible.
 - However, Android and iOS kill apps running in the background is a
```
Comment (missytake) [inline en/help.md:447]:
```suggestion
- That Android and iOS kill apps running in the background is a
```

Comment (missytake) [reply en/help.md:447]: something like this?

**Review by missytake — APPROVED** [review]
But all in all, I think we can also leave it like this, I don't have a reaaally good suggestion.

**Comment by r10s** [comment]
yip, let's leave it as is for now. i think, at some point, the faq needs a larger update anyway.

**Pull request merged**